### PR TITLE
Eventhubs pylint errors

### PR DESCRIPTION
**Pull request by YijunXieMS**
1. Pylint errors except for eventprocessors, which is being developed
2. Some small fixes from preview code reviews

### CODE REVIEW

**Comment by adxsdk6** [comment]
Can one of the admins verify this patch?

File: sdk/eventhub/azure-eventhubs/azure/eventhub/common.py [hunk 1]
```diff
@@ -308,10 +308,10 @@ def try_add(self, event_data):
             raise TypeError('event_data should be type of EventData')
 
         if self._partition_key:
-            if event_data.partition_key and not (event_data.partition_key == self._partition_key):
+            if event_data.partition_key and not event_data.partition_key == self._partition_key:
```
Comment (yunhaoling) [inline sdk/eventhub/azure-eventhubs/azure/eventhub/common.py]: We can make it `event_data.partition_key != self._partition_key` like the one you modified in another piece of code.

File: sdk/eventhub/azure-eventhubs/azure/eventhub/producer.py [hunk 2]
```diff
@@ -229,18 +228,18 @@ def send(self, event_data, partition_key=None, timeout=None):
             wrapper_event_data = event_data
         else:
             if isinstance(event_data, EventDataBatch):  # The partition_key in the param will be omitted.
-                if partition_key and not (partition_key == event_data._partition_key):  # pylint: disable=protected-access
+                if partition_key and not partition_key == event_data._partition_key:  # pylint: disable=protected-access
```
Comment (yunhaoling) [inline sdk/eventhub/azure-eventhubs/azure/eventhub/producer.py]: make it `partition_key != event_data._partition_key`?

Comment (YijunXieMS) [reply sdk/eventhub/azure-eventhubs/azure/eventhub/producer.py]: Changed

File: sdk/eventhub/azure-eventhubs/azure/eventhub/common.py (hunk 1 above)
Comment (YijunXieMS) [reply sdk/eventhub/azure-eventhubs/azure/eventhub/common.py]: changed

**Pull request merged**